joomla/joomla-cms
# [com_finder] Improve disabled smart search content plugin message

#### Summary of Changes

Simple PR to improve the disabled Smart Search Content plugin in com_finder.

Before (the link is to the plugin list with harcoded text search)
![image](https://cloud.githubusercontent.com/assets/9630530/15808299/50ea2426-2b6c-11e6-8f89-a19ae48a71cc.png)

After (the link is now to edit the Smart Search Content plugin)
![image](https://cloud.githubusercontent.com/assets/9630530/15808296/34c4e542-2b6c-11e6-8960-63c99c60fdaf.png)
#### Testing Instructions

Code review, or:
1. Use latest staging
2. Disable Smart Search Content plugin
3. Go to Components -> Smart Search and check the warning message. Click the link, you'll be redirected to the plugin manager list view with a search.
4. Apply patch
5. Go to Components -> Smart Search and check the warning message. Click the link, you'll be redirected to directly edit the smart search content plugin.
